### DISCUSSION TITLE
How can I get the RenderError error content in it after render('index.liquid')?

### DISCUSSION THREAD

**Discussion by maiyuanrdt** [harttle/liquidjs#506] [General]
How can I get the RenderError error content in it after render('index.liquid')?

When some tags or filters are parsed abnormally, they want to get the abnormal content and mix it into the final html result, so that the page can report the error message to the front-end template friendly.

Tried to use catch, it didn't work
render('index.liquid').catch(e => e.message)

try{
renderSync('index.liquid')
} catch(e) {
e.message
}

**Comment by harttle** [comment]
It should work. Could you provide a runnable demo for this problem?

LiquidJS Playground does a similar work, when you visit
https://liquidjs.com/playground.html#eyUgYXNzaWduCg==,e30= it calls render
and catches any render error.
